sapi/cli/php_cli_server.c

```diff
@@ -2421,8 +2421,8 @@ static void php_cli_server_startup_workers(void) {
 	if (php_cli_server_workers_max > 1) {
 		zend_long php_cli_server_worker;
 
-		php_cli_server_workers = calloc(
-			php_cli_server_workers_max, sizeof(pid_t));
+		php_cli_server_workers = safe_emalloc(
+			php_cli_server_workers_max, sizeof(pid_t), 0);
```

Comment:
Two keys here:
1/ > and clear the memory afterwards.
since you re moving from calloc.
2/ You might need a change line 2302.

Reply:
Ok. I have replaced it with `efree`.

Reply:
Indeed. Don t forget the first point :)

Reply:
`ecalloc` is a safe version of `calloc` that checks for multiply overflow. I think it can be used here.

Reply:
Alright, @cmb69 will have a last look in case I missed something. Nice work.